### PR TITLE
Add mikrotik routeros ipv6 neighbor print

### PR DESCRIPTION
**Pull request by k-ribot**
##### ISSUE TYPE
 New Template Pull Request

##### COMPONENT
MikroTik, RouterOs, ipv6 neighbor print

##### SUMMARY
I found examples output in official documentation:
https://wiki.mikrotik.com/wiki/Manual:IPv6/Neighbors

### CODE REVIEW

**Review by jvanderaa — COMMENTED** [review]
Looking good, pending tests.

**Comment by jvanderaa** [comment]
We have some conflicts that have come up.

File: ntc_templates/templates/mikrotik_routeros_ipv6_neighbor_print_without-paging.textfsm [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Value INTERFACE (\S+)
+Value MAC ([0-9a-fA-F]{2}(:[0-9a-fA-F]{2}){5})
+Value STATUS (noarp|incomplete|stale|reachable|delay|probe|failed)
+
+Start
+  ^Flags:.*$$ -> NeighborsTable
+
+NeighborsTable
+  ^\s*${INDEX}\s*(${FLAGS}\s+)?address=${IP}\s+interface=${INTERFACE}\s+(mac-address=${MAC}\s+)?status="${STATUS}"\s*$$ -> Record
+  ^. -> Error
```
Comment (itdependsnetworks) [inline ntc_templates/templates/mikrotik_routeros_ipv6_neighbor_print_without-paging.textfsm:13]:
```suggestion
  ^\s*$$
  ^. -> Error

NeighborsTable
  ^\s*${INDEX}\s*(${FLAGS}\s+)?address=${IP}\s+interface=${INTERFACE}\s+(mac-address=${MAC}\s+)?status="${STATUS}"\s*$$ -> Record
  ^\s*$$
  ^. -> Error
```

Comment (itdependsnetworks) [reply ntc_templates/templates/mikrotik_routeros_ipv6_neighbor_print_without-paging.textfsm:13]: my spacing may be off, but i think you get the idea.

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**